pkg/config/config_template.yaml

```diff
@@ -897,11 +897,10 @@ api_key:
 
 {{- if (eq .OS "windows")}}
   ## @param windows_counter_refresh_interval - number - optional - default: 60
-  ## This controls the interval in seconds for which windows performance counters are refreshed by the agent.
-  ## Refreshes are necessary for the discovery of new counters/countersets and instances for integrations that
-  ## rely on performance counters. Checks include active_directory, aspdotnet, dotnetclr, exchange_server, hyperv,
-  ## iis and windows_performance_counters.
-  ## Refresh can be disabled by setting the interval to 0.
+  ## This controls frequency in seconds of refreshing cached performance counters objects list. It maybe
+  ## useful in rare cases when initialization of a few Agent’s builtin performance counters based checks
```

Comment:
```suggestion
  ## useful in rare cases when initialization of some of the Agent’s builtin performance counters-based checks
```

Reply:
Thank you @sgnn7 and @alai97 for the review. After deliberation with @clarkb7 we have decided to remove its documentation from config template entirely (but not from the code) and leave a comment of similar sort in the code and new PR,

Reply:
Closing the PR to create a new one.